recipes/langchain/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "langchain" %}
+{% set version = "0.0.136" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain-{{ version }}.tar.gz
+  sha256: 6ab89b1d0d350792fba1cfe5f236f97d3484490b7ed61294156a1ddd8b68f187
+
+build:
+  entry_points:
+    - langchain-server = langchain.server:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.1,<4.0
+    - pydantic >=1.0.0,<2.0.0
+    - sqlalchemy >=1.0.0,<2.0.0
+    - requests >=2.0.0,<3.0.0
+    - pyyaml >=5.4.1
+    - numpy >=1.0.0,<2.0.0
+    - openapi-schema-pydantic >=1.2.0,<2.0.0
+    - dataclasses-json >=0.5.7,<0.6.0
+    - tenacity >=8.1.0,<9.0.0
+    - aiohttp >=3.8.3,<4.0.0
+    - async-timeout >=4.0.0,<5.0.0
+  run_constrained:
```

Comment:
I believe these are generated by grayskull based on the package extras/optional packages. It is super nice to have them but we need to keep them updated later! Be careful to always re-generate the recipe with grayskull when updating it on the feedstock.